### PR TITLE
Started githubteacher bio

### LINKED ISSUE #4: Add bio for githubteacher
# Add a bio

Things to include:
- [ ] where I am from
- [ ] favorite food
- [ ] favorite color

@mikejahn

### PR DESCRIPTION
**Pull request by githubteacher**
Resolves #4 

Did I miss anything @jphorec

### CODE REVIEW

File: bio/githubteacher-bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## About GitHub Teacher
+
+- works in San Francisco
+- lives in NC
```
Comment (githubteacher) [inline bio/githubteacher-bio.md]: Where in NC?

**Comment by githubteacher** [comment]
Hey @sdg9 does this look OK?

**Comment by sdg9** [comment]
Looks great

**Comment by githubteacher** [comment]
:+1: Looks good :shipit:

**Pull request merged**